guides/data.livemd

```diff
@@ -0,0 +1,264 @@
+# VegaLite Data
+
+```elixir
+Mix.install([
+  {:explorer, "~> 0.6.1"},
+  {:kino, "~> 0.10.0"},
+  {:vega_lite,
+   git: "https://github.com/cristineguadelupe/vega_lite", branch: "cg-stats", override: true},
+  {:kino_vega_lite, "~> 0.1.9"}
+])
+```
+
+## Introduction
+
+The `VegaLite.Data` module is designed to provide a shorthand API to plot commonly used charts and high-level abstractions for specialized plots.
+
+The API can be combined with the main `VegaLite` module at any level and at any point, providing flexibility to achieve the same results in a more concise way without compromising expressiveness.
+
+Throughout this guide, we will look at how to use the API alone, in combination with the `VegaLite` module, and also show some comparisons between all the possible paths to achieve the same plotting results.
+
+**Limitations**: `VegaLite.Data` relies on internal type inference, and although all options may be overridden, only data that implements the [Table.Reader](https://hexdocs.pm/table/Table.Reader.html) protocol is supported.
+
+For meaningful examples, we will use the *fuels* dataset directly from [Explorer](https://hexdocs.pm/explorer/Explorer.html).
+
+```elixir
+alias Explorer.DataFrame, as: DF
+alias VegaLite, as: Vl
+alias VegaLite.Data
+
+fuels = Explorer.Datasets.fossil_fuels()
+
+data = [
+  %{"category" => "A", "score" => 28},
+  %{"category" => "B", "score" => 50},
+  %{"category" => "C", "score" => 34},
+  %{"category" => "D", "score" => 42},
+  %{"category" => "E", "score" => 39}
+]
+```
+
+## Chart - the shorthand api
+
+`VegaLite.Data.chart/3` and `VegaLite.Data.chart/4` are the shorthand API. We will use these functions to get quick and concise plots. It's shine for plots that don't require a lot of configuration or customization.
+
+`VegaLite.Data.chart/3` takes 3 arguments: the data, the mark and a list of fields to be encoded while `VegaLite.Data.chart/4` works similarly, but takes a valid `VegaLite` specification as the first argument.
+
+```elixir
+# A simple bar plot using the shorthand api
+Data.chart(data, :bar, x: "category", y: "score")
+```
+
+```elixir
+# The same chart without the shorthand api
+Vl.new()
+|> Vl.data_from_values(data)
+|> Vl.mark(:bar)
+|> Vl.encode_field(:y, "score", type: :quantitative)
+|> Vl.encode_field(:x, "category", type: :nominal)
+```
+
+Plotting a simple chart is a breeze! As we can see from the comparison above, the code becomes much leaner and handleable. However, the API also accepts a list of options for each argument, allowing more complex results.
+
+```elixir
+# A line plot with point: true
+
+# Without the shorthand api
+Vl.new()
+|> Vl.data_from_values(fuels, only: ["total", "solid_fuel"])
+|> Vl.mark(:line, point: true)
+|> Vl.encode_field(:x, "total", type: :quantitative)
+|> Vl.encode_field(:y, "solid_fuel", type: :quantitative)
+
+Data.chart(fuels, [type: :line, point: true], x: "total", y: "solid_fuel")
```

Comment:
In the first example, you have one Elixir code snippet per chart, with the `Data` version coming first. Should we make them consistent throughout the guide?